config/zones.json

```diff
@@ -1524,12 +1531,39 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1754,
+      "biomass": 450
+    },
     "parsers": {
       "production": "US_NY.fetch_production"
     },
     "flag_file_name": "us.png",
     "contributors": [
-      "https://github.com/systemcatch"
+      "https://github.com/systemcatch",
+      "https://github.com/hanshyde"
+    ]
+  },
+  "US-PJM": {
+    "_comment": "Installed capacity in PJM RTO effective June 1, 2017 - http://www.pjm.com/-/media/markets-ops/ops-analysis/capacity-by-fuel-type-2017.ashx?la=en",
```

Comment:
No need for this comment as we already have it in the readme (duplicate information is hard to maintain)